.github/workflows/wheels.yml

```diff
@@ -209,9 +203,10 @@ jobs:
           # build and test the wheel
           auto-update-conda: true
           python-version: "3.10"
+          miniconda-version: "latest"
```

Comment:
I get [errors on the x86 runners](https://github.com/thalassemia/scipy/actions/runs/8699601592/job/23858459934) about a missing `conda` base environment without this.